modules/administration-guide/pages/applying-the-bitbucket-application-link-secret.adoc

```diff
@@ -0,0 +1,58 @@
+:_content-type: PROCEDURE
+:description: Applying the Bitbucket application link Secret
+:keywords: applying-bitbucket, apply-the-bitbucket, apply-bitbucket, apply-secret, applying-secret, apply-a-secret, applying-a-secret, bitbucket, bitbucket-application, bitbucket-app, bitbucket-application-link, bitbucket-app-link, bitbucket-secret, secret
+:navtitle: Applying the Bitbucket application link Secret
+// :page-aliases:
+
+[id="applying-the-bitbucket-application-link-secret_{context}"]
+= Applying the Bitbucket application link Secret
+
+Enable the users of your {prod-short} instance to access a private Git repository that is hosted using Bitbucket:
+
+.Prerequisites
+* The repository owner has xref:user-guide:setting-up-the-bitbucket-application-link.adoc[set up the Bitbucket application link].
+* The repository owner has given you the (1) *privatepkcs8-stripped.pem* file and (2) *bitbucket_consumer_key* file.
```

Comment:
This doesn't necessarily have to be done by the private repository's owner, its more typical for an admin to set tup the application link (same goes for GitHub Oauth app and Gitlab application ID), so I suggest we rephrase this part

Reply:
Maybe something like:
```
* The xref:user-guide:setting-up-the-bitbucket-application-link.adoc[Bitbucket application link] has been set up.
```

or something similar to that?

Reply:
Yes, I'm rewriting all the prerequisites in this PR and I'm moving the files from the UG to the AG.